### ISSUE TITLE
Remove getAbundanceSample and getAbundanceFeature

### ISSUE THREAD

**Issue by antagomir** [microbiome/mia#409]
Do we really need those two functions? We can just call `tse[1,]` and `tse[, "ID2"]` etc. for the same.

**Comment by microsud** [comment]
If I remember correctly, we decided to include this to make it intuitive for people transitioning from phyloseq. tse[1,] and tse[, "ID2"] will not work. It should be https://github.com/microbiome/mia/blob/531768d893c812fc8d3e1dcbe91fdce7d2144b60/vignettes/mia.Rmd#L288
```
library(mia)
data("GlobalPatterns")
assay(GlobalPatterns, "counts")['522457',]
assay(GlobalPatterns, "counts")[,'CC1']
```

**Comment by antagomir** [comment]
Ah true. Or also:

```
library(mia)
data("GlobalPatterns")
assay(GlobalPatterns['522457',], "counts")
assay(GlobalPatterns[,'CC1'], "counts")
```

If the only motivation to maintain the rarely used functions  `getAbundanceSample` and `getAbundanceFeature` then I am not sure if it is really critical to keep these. From maintenance perspective it would be better to get rid of these I think. Unless there is a real need. If someone gets to work with TreeSE for real I expect that they won't use these anyway?

**Comment by microsud** [comment]
Yes, if someone is using treeSE for some time, they will not use it. Shall I  make a PR to remove this?

**Comment by antagomir** [comment]
Yes I would suggest so. 

In principle these should be deprecated but perhaps there is no real need and could be just removed?

**Comment by microsud** [comment]
To make changes, shall I pull the main branch or any other recent branch?

**Comment by antagomir** [comment]
Pull the main branch, and then open the PR from a new branch in this repo, or from another fork.

**Comment by microsud** [comment]
Thanks! Looking for suggestion/opinion 
I am running checks and see lot of warning for deprecated function names in the package that have not been updated in the **tests** section. Shall I fix these here?

```
package 'dplyr' was built under R version 4.2.2
Backtrace:
 1. base::library(dplyr)
      at test-10makeTreeSEFromBiom.R:8:4
 2. base (local) testRversion(pkgInfo, package, pkgpath)
── Warning ('test-10makeTreeSEFromBiom.R:10'): makeTreeSEFromBiom ──────────────
The following artifacts: '"' were cleaned from 
                    rowData.
Backtrace:
 1. mia::makeTreeSEFromBiom(...)
      at test-10makeTreeSEFromBiom.R:10:4
 3. mia:::.detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
 4. mia:::.clean_from_artifacts(x, pattern)
── Warning ('test-10makeTreeSEFromBiom.R:27'): makeTreeSEFromBiom ──────────────
The following artifacts: '"' were cleaned from 
                    rowData.
Backtrace:
 1. mia::makeTreeSEFromBiom(...)
      at test-10makeTreeSEFromBiom.R:27:4
 3. mia:::.detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
 4. mia:::.clean_from_artifacts(x, pattern)
── Warning ('test-10makeTreeSEFromBiom.R:36'): makeTreeSEFromBiom ──────────────
The following artifacts: '"' were cleaned from 
                    rowData.
Backtrace:
 1. mia::makeTreeSEFromBiom(...)
      at test-10makeTreeSEFromBiom.R:36:4
 3. mia:::.detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
 4. mia:::.clean_from_artifacts(x, pattern)
── Warning ('test-10makeTreeSEFromBiom.R:46'): makeTreeSEFromBiom ──────────────
The '\*|\?' provided at 'clean.taxa.names' were
                    not found in rowData.
Backtrace:
 1. mia::makeTreeSEFromBiom(...)
      at test-10makeTreeSEFromBiom.R:46:4
 3. mia:::.detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
── Warning ('test-10makeTreeSEFromBiom.R:54'): makeTreeSEFromBiom ──────────────
The following artifacts: '"' were cleaned from 
                    rowData.
Backtrace:
 1. mia::makeTreeSEFromBiom(...)
      at test-10makeTreeSEFromBiom.R:54:4
 3. mia:::.detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
 4. mia:::.clean_from_artifacts(x, patterns)
── Warning ('test-10makeTreeSEFromBiom.R:70'): makeTreeSEFromBiom ──────────────
The following artifacts: '"' were cleaned from 
                    rowData.
Backtrace:
 1. mia::makeTreeSEFromBiom(...)
      at test-10makeTreeSEFromBiom.R:70:4
 3. mia:::.detect_taxa_artifacts_and_clean(feature_data, clean.taxa.names)
 4. mia:::.clean_from_artifacts(x, patterns)
── Warning ('test-2merge.R:68'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(x, xr, xrl), FUN_check_x)
      at test-2merge.R:68:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:68'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(x, xr, xrl), FUN_check_x)
      at test-2merge.R:68:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:68'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(x, xr, xrl), FUN_check_x)
      at test-2merge.R:68:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:69'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(x, xr, xrl), FUN_check_x, archetype = 2)
      at test-2merge.R:69:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:69'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(x, xr, xrl), FUN_check_x, archetype = 2)
      at test-2merge.R:69:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:69'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(x, xr, xrl), FUN_check_x, archetype = 2)
      at test-2merge.R:69:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:84'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(xtse), FUN_check_x)
      at test-2merge.R:84:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. mia (local) .local(x, f, archetype, ...)
 6. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:85'): merge ─────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. base::lapply(list(xtse), FUN_check_x, archetype = 2)
      at test-2merge.R:85:4
 2. mia (local) FUN(X[[i]], ...)
 4. mia::mergeFeatures(x, f, archetype)
 5. mia (local) .local(x, f, archetype, ...)
 6. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:102'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. mia::mergeFeatures(tse, f = rowData(tse)$group2, mergeTree = TRUE)
      at test-2merge.R:102:4
 2. mia::mergeFeatures(tse, f = rowData(tse)$group2, mergeTree = TRUE)
 3. mia (local) .local(x, f, archetype, ...)
 4. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:103'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. mia::mergeFeatures(tse, f = rowData(tse)$group2, mergeTree = FALSE)
      at test-2merge.R:103:4
 2. mia::mergeFeatures(tse, f = rowData(tse)$group2, mergeTree = FALSE)
 3. mia (local) .local(x, f, archetype, ...)
 4. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:104'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. mia::mergeFeatures(...)
      at test-2merge.R:104:4
 2. mia::mergeFeatures(...)
 3. mia (local) .local(x, f, archetype, ...)
 4. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:106'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. mia::mergeFeatures(tse, f = rowData(tse)$group2, mergeTree = TRUE)
      at test-2merge.R:106:4
 2. mia::mergeFeatures(tse, f = rowData(tse)$group2, mergeTree = TRUE)
 3. mia (local) .local(x, f, archetype, ...)
 4. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:113'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. testthat::expect_equal(...)
      at test-2merge.R:113:4
 5. mia::mergeFeatures(tse, f = rowData(tse)$group2)
 6. mia (local) .local(x, f, archetype, ...)
 7. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:116'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. mia::mergeFeatures(tse, f = rowData(tse)$group, mergeTree = TRUE)
      at test-2merge.R:116:4
 2. mia::mergeFeatures(tse, f = rowData(tse)$group, mergeTree = TRUE)
 3. mia (local) .local(x, f, archetype, ...)
 4. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2merge.R:117'): merge ────────────────────────────────────────
'mergeRows' is deprecated.
Use 'mergeFeatures' instead.
See help("Deprecated") and help("Now mergeRows is deprecated. Use mergeFeatures instead.-deprecated").
Backtrace:
 1. mia::mergeFeatures(tse, f = rowData(tse)$group, mergeTree = FALSE)
      at test-2merge.R:117:4
 2. mia::mergeFeatures(tse, f = rowData(tse)$group, mergeTree = FALSE)
 3. mia (local) .local(x, f, archetype, ...)
 4. base::.Deprecated(old = "mergeRows", new = "mergeFeatures", "Now mergeRows is deprecated. Use mergeFeatures instead.")
── Warning ('test-2taxonomy.R:126'): taxonomy ──────────────────────────────────
package 'RSQLite' was built under R version 4.2.2
Backtrace:
 1. base::library(DECIPHER)
      at test-2taxonomy.R:126:4
 2. base::.getRequiredPackages2(pkgInfo, quietly = quietly)
 3. base::library(...)
 4. base (local) testRversion(pkgInfo, package, pkgpath)
── Warning ('test-3agglomerate.R:30'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. testthat::expect_error(...)
      at test-3agglomerate.R:30:4
 7. mia::mergeFeaturesByRank(xtse, "", na.rm = FALSE)
 8. mia (local) .local(x, ...)
 9. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:32'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. testthat::expect_error(...)
      at test-3agglomerate.R:32:4
 7. mia::mergeFeaturesByRank(xtse, "Family", na.rm = "")
 8. mia (local) .local(x, ...)
 9. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:34'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. testthat::expect_error(...)
      at test-3agglomerate.R:34:4
 7. mia::mergeFeaturesByRank(xtse, "Family", na.rm = FALSE, agglomerateTree = "")
 8. mia (local) .local(x, ...)
 9. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:38'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. testthat::expect_error(...)
      at test-3agglomerate.R:38:4
 7. mia::mergeFeaturesByRank(xtse2, "Family", na.rm = FALSE)
 8. mia (local) .local(x, ...)
 9. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:41'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(xtse, "Family", na.rm = FALSE)
      at test-3agglomerate.R:41:4
 2. mia::mergeFeaturesByRank(xtse, "Family", na.rm = FALSE)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:43'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(xtse, "Phylum", na.rm = FALSE)
      at test-3agglomerate.R:43:4
 2. mia::mergeFeaturesByRank(xtse, "Phylum", na.rm = FALSE)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:46'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(xtse, "Family", na.rm = TRUE)
      at test-3agglomerate.R:46:4
 2. mia::mergeFeaturesByRank(xtse, "Family", na.rm = TRUE)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:49'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(xtse, "Family", na.rm = FALSE)
      at test-3agglomerate.R:49:4
 2. mia::mergeFeaturesByRank(xtse, "Family", na.rm = FALSE)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:52'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(xtse, "Phylum")
      at test-3agglomerate.R:52:4
 2. mia::mergeFeaturesByRank(xtse, "Phylum")
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:55'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(xtse, "Family")
      at test-3agglomerate.R:55:4
 2. mia::mergeFeaturesByRank(xtse, "Family")
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:58'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. testthat::expect_equal(...)
      at test-3agglomerate.R:58:4
 5. mia::mergeFeaturesByRank(xtse, "Family")
 6. mia (local) .local(x, ...)
 7. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:63'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. testthat::expect_equal(...)
      at test-3agglomerate.R:63:4
 6. mia::mergeFeaturesByRank(enterotype, "Genus")
 7. mia (local) .local(x, ...)
 8. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:69'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(se, rank = "Family")
      at test-3agglomerate.R:69:4
 2. mia::mergeFeaturesByRank(se, rank = "Family")
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-3agglomerate.R:73'): agglomerate ─────────────────────────────
'agglomerateByRank' is deprecated.
Use 'mergeFeaturesByRank' instead.
See help("Deprecated") and help("Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByRank(se, rank = "Family", mergeTree = TRUE)
      at test-3agglomerate.R:73:4
 2. mia::mergeFeaturesByRank(se, rank = "Family", mergeTree = TRUE)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-5prevalence.R:330'): mergeFeaturesByPrevalence ───────────────
'agglomerateByPrevalence' is deprecated.
Use 'mergeFeaturesByPrevalence' instead.
See help("Deprecated") and help("Now agglomerateByPrevalence is deprecated. Use mergeFeaturesByPrevalence instead.-deprecated").
Backtrace:
 1. testthat::expect_error(...)
      at test-5prevalence.R:330:4
 7. mia::mergeFeaturesByPrevalence(GlobalPatterns, other_label = TRUE)
 8. mia (local) .local(x, ...)
 9. base::.Deprecated(...)
── Warning ('test-5prevalence.R:332'): mergeFeaturesByPrevalence ───────────────
'agglomerateByPrevalence' is deprecated.
Use 'mergeFeaturesByPrevalence' instead.
See help("Deprecated") and help("Now agglomerateByPrevalence is deprecated. Use mergeFeaturesByPrevalence instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByPrevalence(GlobalPatterns)
      at test-5prevalence.R:332:4
 2. mia::mergeFeaturesByPrevalence(GlobalPatterns)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-5prevalence.R:336'): mergeFeaturesByPrevalence ───────────────
'agglomerateByPrevalence' is deprecated.
Use 'mergeFeaturesByPrevalence' instead.
See help("Deprecated") and help("Now agglomerateByPrevalence is deprecated. Use mergeFeaturesByPrevalence instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByPrevalence(...)
      at test-5prevalence.R:336:4
 2. mia::mergeFeaturesByPrevalence(...)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-5prevalence.R:346'): mergeFeaturesByPrevalence ───────────────
'agglomerateByPrevalence' is deprecated.
Use 'mergeFeaturesByPrevalence' instead.
See help("Deprecated") and help("Now agglomerateByPrevalence is deprecated. Use mergeFeaturesByPrevalence instead.-deprecated").
Backtrace:
 1. mia::mergeFeaturesByPrevalence(...)
      at test-5prevalence.R:346:4
 2. mia::mergeFeaturesByPrevalence(...)
 3. mia (local) .local(x, ...)
 4. base::.Deprecated(...)
── Warning ('test-5prevalence.R:352'): mergeFeaturesByPrevalence ───────────────
'agglomerateByPrevalence' is deprecated.
Use 'mergeFeaturesByPrevalence' instead.
See help("Deprecated") and help("Now agglomerateByPrevalence is deprecated. Use mergeFeaturesByPrevalence instead.-deprecated").
Backtrace:
 1. testthat::expect_equal(...)
      at test-5prevalence.R:352:4
 5. mia::mergeFeaturesByPrevalence(...)
 6. mia (local) .local(x, ...)
 7. base::.Deprecated(...)
── Warning ('test-5runDPCoA.R:3'): runDPCoA ────────────────────────────────────
package 'ade4' was built under R version 4.2.2
Backtrace:
 1. testthat::skip_if_not(require("ade4", quietly = TRUE))
      at test-5runDPCoA.R:3:4
 3. base::require("ade4", quietly = TRUE)
 8. base::library(...)
 9. base (local) testRversion(pkgInfo, package, pkgpath)

══ Failed tests ════════════════════════════════════════════════════════════════
── Failure ('test-0utilites.R:96'): getAbundanceFeature/getAbundanceSample ─────
`mia:::.check_feature_sample_ids()` threw an error with unexpected message.
Expected match: "argument \"id\" is missing"
Actual message: "object '.check_feature_sample_ids' not found"
Backtrace:
    ▆
 1. └─testthat::expect_error(mia:::.check_feature_sample_ids(), "argument \"id\" is missing") at test-0utilites.R:96:4
 2.   └─testthat:::quasi_capture(...)
 3.     ├─testthat (local) .capture(...)
 4.     │ └─base::withCallingHandlers(...)
 5.     └─rlang::eval_bare(quo_get_expr(.quo), quo_get_env(.quo))
── Failure ('test-0utilites.R:98'): getAbundanceFeature/getAbundanceSample ─────
`mia:::.check_feature_sample_ids("test")` threw an error with unexpected message.
Expected match: "argument \"names\" is missing"
Actual message: "object '.check_feature_sample_ids' not found"
Backtrace:
    ▆
 1. └─testthat::expect_error(...) at test-0utilites.R:98:4
 2.   └─testthat:::quasi_capture(...)
 3.     ├─testthat (local) .capture(...)
 4.     │ └─base::withCallingHandlers(...)
 5.     └─rlang::eval_bare(quo_get_expr(.quo), quo_get_env(.quo))
── Error ('test-0utilites.R:100'): getAbundanceFeature/getAbundanceSample ──────
Error in `eval_bare(expr, quo_get_env(quo))`: object '.check_feature_sample_ids' not found
Backtrace:
    ▆
 1. └─testthat::expect_null(...) at test-0utilites.R:100:4
 2.   └─testthat::quasi_label(enquo(object), label, arg = "object")
 3.     └─rlang::eval_bare(expr, quo_get_env(quo))

[ FAIL 3 | WARN 43 | SKIP 3 | PASS 902 ]
Error: Test failures
Execution halted
```

**Comment by antagomir** [comment]
Yes, that would be great! Was omitted by mistake in the recent PR I guess.

**Comment by antagomir** [comment]
Closed via #413

**Comment by TuomasBorman** [comment]
This is still open, but I think this can be closed. (If not, reopen)
